orion/main.go

```diff
@@ -8,62 +8,70 @@ import (
+	"github.com/ahsu1230/mathnavigatorSite/orion/controllers/announce"
 	"github.com/ahsu1230/mathnavigatorSite/orion/controllers/achieve"
 	"github.com/ahsu1230/mathnavigatorSite/orion/middlewares"
 	"github.com/ahsu1230/mathnavigatorSite/orion/database"
 )
 
 func main() {
-	fmt.Println("Orion service starting...")
+  fmt.Println("Orion service starting...")
 
-	config := middlewares.RetrieveConfigurations()
-	fmt.Println("Building server in mode: ", config.App.Build)
+  config := middlewares.RetrieveConfigurations()
+  fmt.Println("Building server in mode: ", config.App.Build)
 
-	fmt.Println("Connecting to DB...")
-	configDb := config.Database
-	database.OpenDb(configDb.Host, configDb.Port,
-		configDb.Username, configDb.Password)
-	fmt.Println("Performing DB Migrations...")
-	database.Migrate()
+  fmt.Println("Connecting to DB...")
+  configDb := config.Database
+  database.OpenDb(configDb.Host, configDb.Port,
+    configDb.Username, configDb.Password)
+  fmt.Println("Performing DB Migrations...")
+  database.Migrate()
 
-	fmt.Println("Setting up Router...")
-	router := gin.Default()
+  fmt.Println("Setting up Router...")
+  router := gin.Default()
 
-	fmt.Println("Setting up Middlewares...")
+  fmt.Println("Setting up Middlewares...")
 
-	// CORS middleware
-	configCors := middlewares.CreateCorsConfig(config)
-	router.Use(cors.New(configCors))
+  // CORS middleware
+  configCors := middlewares.CreateCorsConfig(config);
+  router.Use(cors.New(configCors))
 
-	// Webpage Routers
-	// router.Use(static.Serve("/", static.LocalFile("./sites/home", true)))
-	router.Use(static.Serve("/", static.LocalFile("./sites/admin", true)))
+  // Webpage Routers
+  // router.Use(static.Serve("/", static.LocalFile("./sites/home", true)))
+  router.Use(static.Serve("/", static.LocalFile("./sites/admin", true)))
 
-	// API Routers
-	apiPrograms := router.Group("/api/programs/")
-	{
-		apiPrograms.GET("/v1/all", programs.GetPrograms)
-		apiPrograms.POST("/v1/create", programs.CreateProgram)
-		apiPrograms.GET("/v1/program/:programId", programs.GetProgram)
-		apiPrograms.POST("/v1/program/:programId", programs.UpdateProgram)
-		apiPrograms.DELETE("/v1/program/:programId", programs.DeleteProgram)
-	}
-	// apiClasses := router.Group("api/classes/")
-	// apiLocations := router.Group("api/locations/")
-	// apiAnnounce := router.Group("api/announce/")
-	apiAchieve := router.Group("api/achievements/")
+  // API Routers
+  apiPrograms := router.Group("/api/programs/")
+  {
+    apiPrograms.GET("/v1/all", programs.GetPrograms)
+    apiPrograms.POST("/v1/create", programs.CreateProgram)
+    apiPrograms.GET("/v1/program/:programId", programs.GetProgram)
+    apiPrograms.POST("/v1/program/:programId", programs.UpdateProgram)
+    apiPrograms.DELETE("/v1/program/:programId", programs.DeleteProgram)
+  }
+  // apiClasses := router.Group("api/classes/")
+  // apiLocations := router.Group("api/locations/")
+  apiAnnounce := router.Group("api/announce/")
```

Comment:
Miscommunication on my part. Change this back to `api/announcements` here.
For the other endpoints below, since we are operating on one announcement at a time, those should stay as `/v1/announce/...`.